data/137/682/640/3/1376826403.geojson

```diff
@@ -39,8 +51,11 @@
         85667981
     ],
     "wof:breaches":[],
+    "wof:concordances":{
+        "wk:page":"Al_Dhafra,_Abu_Dhabi"
```

Comment:
We should bring in the Wikidata ID of [Q470386](https://www.wikidata.org/wiki/Q4703864) which includes a few more name translations.